workers/unity/Packages/com.improbable.gdk.deploymentlauncher/.DeploymentLauncher/README.md

```diff
@@ -0,0 +1,6 @@
+This project exists in folder prefixed with a '.' so that Unity ignores it at script compile time, but it can still be
```

Comment:
```suggestion
This project exists in a folder prefixed with a '.' so that Unity ignores it at script compile time, but it can still be
```

and do we need the linebreak here?

Reply:
I guess its nice if you don't have an Editor with line wraps?